json-rpc/types/src/views.rs

```diff
@@ -68,9 +69,63 @@ pub enum AccountRoleView {
         received_mint_events_key: BytesView,
         compliance_key_rotation_events_key: BytesView,
         base_url_rotation_events_key: BytesView,
+        preburn_queues: Option<Vec<PreburnQueueView>>,
```

Comment:
@xli I wound up having to change this field to be nullable in order to keep backwards compatibility (see the cluster test failure -- new client, but talking to an older json-rpc server). Let me know if the change looks good to you.

Reply:
Yeah, that's correct. Any fields added to an existing struct or variant would need to be nullable. for compatibility reasons.